src/cookbooks/glam.md

```diff
@@ -1,10 +1,10 @@
 # Introduction to GLAM
 
-GLAM was built to help Mozillians answer their data questions without needing data analysis or coding skills. It contains a visualization layer meant to answer most "easy" questions about how a probe or metric has changed over build ids and releases.
+GLAM was built to help Mozillians answer most "easy" questions about how a probe or metric has changed over build ids and releases.
```

Comment:
Is it for Desktop or all browsers?
When I read it I think "easy" is a bit vague, maybe easier to just remove it / give one or two examples?

Reply:
Reading below I see is for all apps, so probably just good to mention it in the intro. Your call.

Reply:
Thanks! That whole paragraph was a little vague.